packages/taiko-client/prover/init.go

```diff
@@ -163,99 +163,91 @@ func (p *Prover) initProofSubmitters(
 	return p.initPacayaProofSubmitter(txBuilder)
 }
 
-// initPacayaProofSubmitter initializes the proof submitter from the non-zero verifier addresses in protocol.
+// initPacayaProofSubmitter initializes the proof submitters from the non-zero verifier addresses set in protocol.
```

Comment:
```suggestion
// initPacayaProofSubmitter initializes the proof submitter from the non-zero verifier addresses set in protocol.
```
Only one submiter existes for Pacaya fork